src/styles/index.scss

```diff
@@ -1,3 +1,124 @@
+* {
+  font-size: 14px;
+  text-decoration: none;
+  box-sizing: border-box;
+}
+@import 'vars';
+
+html{
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+  font-style: normal;
+}
+
 body {
   margin: 0;
 }
+
+p {
+  margin: 0;
+}
+
+.card {
+  position: relative;
+  width: 200px;
+  height: 406px;
```

Comment:
Do not hard code height
```suggestion

```

Reply:
I was suggested to fixed height in chat by mentor

Reply:
not fixed